examples/ui-demo/src/components/small-cards/Badge.tsx

```diff
@@ -0,0 +1,13 @@
+export function Badge({
+  text,
+  className,
+}: {
+  text: string;
+  className?: string;
+}) {
+  return (
+    <div className={`px-2 py-1 font-semibold rounded-md text-xs ${className}`}>
```

Comment:
nit (non-blocking): would prefer to use the `cn` util here so that the base classes could be properly override if a conflicting class is passed in.